indico/htdocs/sass/modules/_users.scss

```diff
@@ -78,10 +78,14 @@
     display: inline-block;
     font-size: 0.95em;
     padding: 0.2em 0.5em;
-    margin-bottom: 0.1em;
+    margin-bottom: 0.5em;
```

Comment:
We don't want to increase `margin-bottom` in all cases. We are using `.event-user` in other places, like for instance in meeting pages ([example](https://indico.cern.ch/event/588588/)).